.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Install Toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2022-04-20
+          toolchain: nightly-2022-05-15
```

Comment:
@1xstj I think this will get overwritten because of the [rust-toolchain.toml](https://github.com/webb-tools/dkg-substrate/blob/master/rust-toolchain.toml) in root